test/rules/named-spy.js

```diff
@@ -19,6 +19,7 @@ eslintTester.run('named-spy', rule, {
     // Correct use of named spies
     'var onSuccess = jasmine.createSpy("onSuccess")',
     'onSuccess = jasmine.createSpy("onSuccess")',
+    'onSuccess(jasmine.createSpy("callback"))',
```

Comment:
@burabure sure thing, I've added a comment to make it more specific - if you meant anything else just let me know :)